nissan_2017.dbc

```diff
@@ -45,8 +45,9 @@ BO_ 645 WheelspeedRear: 8 XXX
  SG_ RR : 7|16@0+ (0.00555,0) [0|65535] "KPH" XXX
  SG_ RL : 23|16@0+ (0.00555,0) [0|65535] "KPH" XXX
 
-BO_ 768 STEER_TORQUE: 8 XXX
- SG_ STEERING_TOURQUE : 0|7@1+ (1,0) [0|127] "" XXX
+BO_ 768 STEER_TORQUE: 2 XXX
+ SG_ STEERING_TORQUE : 0|7@1+ (1,0) [0|127] "" XXX
```

Comment:
can we keep this is big endian? Basically:

SG_ STEERING_TORQUE : 6|7@0+ (1,0) [0|127] "" XXX